bigflow/clear_methods.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
```

Comment:
I think 18 lines of code, used only in 1 module, is not enough to make a new module

Reply:
Sorry, now I see that you use it in another modules, but still I think it should be in the build module, because it clears build leftovers.

Reply:
It cannot be in build modules because build is imported in setup.py and setup.py imports xmlrunner. At the moment of using setup.py xmlrunner is not installed yet so we get ImportError